supervisor/api/addons.py

```diff
@@ -255,12 +257,13 @@ async def info(self, request: web.Request) -> Dict[str, Any]:
             ATTR_INGRESS_URL: None,
             ATTR_INGRESS_PORT: None,
             ATTR_INGRESS_PANEL: None,
+            ATTR_WATCHDOG: None,
         }
 
         if isinstance(addon, Addon) and addon.is_installed:
             data.update(
                 {
-                    ATTR_STATE: await addon.state(),
+                    ATTR_STATE: addon.state,
```

Comment:
No need for this line with the change above.

Reply:
because of it :)